contracts/modules/capital/Pool.sol

```diff
@@ -93,9 +92,6 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
     priceFeedOracle = IPriceFeedOracle(_priceOracle);
     swapOperator = _swapOperator;
 
-    // [todo] After this contract is deployed it might be worth modifying upgradeCapitalPool to
```

Comment:
I'll create an issue to track this.

Reply:
Let's keep the todo but add the issue id.